argilla/docs/tutorials/image_preference.ipynb

```diff
@@ -0,0 +1,455 @@
+{
```

Comment:
<p>I haven't checked the ImageField usage, so the base64 string is the only way of passing images, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5413/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
You could pass a data uri (base64 + a reference) or a url to an image that's accessible from the argilla server.